applications/ws-fund-ner/README.md

```diff
@@ -0,0 +1,25 @@
+# ws-funder-ner@1.0.0
+
+## Build
+
+Required: initialize environment variables:
+
+- `WEBDAV_URL`
+- `WEBDAV_LOGIN`
+- `WEBDAV_PASSWORD`
+
+> **Remember:** when using a webdav remote, the URL's protocol is `webdav`.
+> **Remember:** Do not forget to export these environment variables.
+
+Before calling `npm run build` or `npm start`.
+
+> 📗 Suggestion: put them in `.env` file and don't forget to run `source .env`
+> before calling `npm run build` or `npm start`.
+
+## Test
+
+```bash
+curl -X POST "http://localhost:31976/v1/funderTagger/funderTagger" \
```

Comment:
This has more to do with <https://gitbucket.inist.fr/tdm/web-services/tree/master/fund-ner>, but this route is a bit complicated.

I'd prefer `v1/funderTagger`, or even `v1/tagger`, as the domain name will include `funder-ner`.
<https://funder-ner.services.istex.fr/v1/tagger> should be enough, shouldn't it?.

Reply:
I changed path in both gitbucket and github repository to v1/tagger .